hacs.json

```diff
@@ -3,5 +3,6 @@
   "content_in_root": false,
   "zip_release": true,
   "filename": "alexa_media.zip",
-  "homeassistant": "2024.2.0b0"
+  "homeassistant": "2024.2.0b0",
+  "hide_default_branch": true
```

Comment:
Why? We always offer dev but it's only for those who want cutting edge.

Reply:
I was seeing this when selecting redownload of my repository and was thinking there was an error:
![image](https://github.com/user-attachments/assets/3414d3f5-9082-4acb-a948-cabfb54e7d43)
as the "official" version doesn't show me dev and Iwas attempting to hid it.  Another NOOB error. My apologies. I will revert it...

Reply:
I was wondering if you've seen [https://github.com/alandtse/alexa_media_player/issues/2293](https://github.com/alandtse/alexa_media_player/issues/2293)>
There seems to be a handful of users for whom adding `"importlib-metadata==6.8.0"` to manfest.json resolves their problem...
```
"requirements": ["alexapy==1.27.10", "packaging>=20.3", "wrapt>=1.14.0", "importlib-metadata==6.8.0"],
```
I'm thinking it might possibly be due to another integration upsetting the apple cart for alexapy? That or they somehow have a different environment than mine? but I'm just guessing.  Maybe you have some idea what might be going on?

Reply:
No idea but it'd have to be a different PR. Let's get this in before you consider separate issues.  Please address the hassfest errors. It's required for HA.

Reply:
I created a new branch, published v4.10.3.1 with a modified manifest.json and 1 user updated to it and it resolved his maintainer error...
```
File "/usr/local/lib/python3.12/site-packages/alexapy/init.py", line 41, in
maintainer = metadata["maintainer"]
```

Reply:
Done. Thanks